crown-apparel/vm_deploy.sh

```diff
@@ -71,20 +67,20 @@ then
   echo -e "\n==== Crownapp VM present ====\n"
 else 
   echo -e "\n==== Creating crownapp VM ====\n"
-multipass launch --cpus 4 --mem 7G --disk 50G --name crownapp --cloud-init cloud-config.yaml
+multipass launch --cpus 4 --memory 7G --disk 50G --name crownapp --cloud-init cloud-config.yaml jammy
 fi 
 
 # Copy necessary files to production deploy using Rsync
 echo -e "\n==== Transferring files to VM ====\n"
 rsync -av -e "ssh -o StrictHostKeyChecking=no -i ./id_ed25519" --delete --exclude={'crownapp.service','.netlify*','build','node_modules','.git','.gitignore','id_ed25519*','cloud-config.yaml'} $(pwd) jason@$(multipass info crownapp |  grep '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}' | awk '  {print $2}'):/home/jason 
```

Comment:
You can use the shell environment variable $HOME which will resolve to the path to your home directory. For example `echo $HOME` would be `/home/jason` if the user is jason.